huaweicloud/services/acceptance/cfw/resource_huaweicloud_cfw_lts_log_test.go

```diff
@@ -0,0 +1,173 @@
+package cfw
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func getLtsLogFunc(cfg *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	region := acceptance.HW_REGION_NAME
+
+	var (
+		httpUrl = "v1/{project_id}/cfw/logs/configuration"
+		product = "cfw"
+	)
+	client, err := cfg.NewServiceClient(product, region)
+	if err != nil {
+		return nil, fmt.Errorf("error creating CFW client: %s", err)
+	}
+
+	path := client.Endpoint + httpUrl
+	path = strings.ReplaceAll(path, "{project_id}", client.ProjectID)
+	path += fmt.Sprintf("?fw_instance_id=%s", state.Primary.ID)
+
+	opt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+	}
+	resp, err := client.Request("GET", path, &opt)
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving lts log configuration: %s", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("error retrieving CFW lts log configuration: %s", err)
```
and fix other places

Reply:
fixed